spec/test_app/README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
Can you put something in this README about the purpose of this app?

It doesn't need to be very detailed, just a sentence or two is probably enough.

Reply:
Sure